src/report/business-reports/receivables.scm

```diff
@@ -50,7 +51,7 @@
       "w" (N_ "The receivables account you wish to examine.") 
       #f #f (list ACCT-TYPE-RECEIVABLE)))
```

Comment:
And this is the spot for the internal option for the receivables report. The option should be exactly the same as what you added in the payables report, except this one will have #t as default value.